docs/layers.md

```diff
@@ -568,6 +568,9 @@ _TIP: Some `landuse` features only exist as point features in OpenStreetMap. Fin
 * `works`
 * `zoo`
 
+##### Wetland `kind_detail` values:
+
+The value of the OpenStreetMap `wetland` tag.
```

Comment:
We should list common values here in a-z alphabetical sorting.

Reply:
✔️